$NetBSD: patch-meson.build,v 1.1 2026/07/20 07:10:35 wiz Exp $

deps needs to be initialized before it can be modified.

--- meson.build.orig	2026-07-20 07:08:14.949369171 +0000
+++ meson.build
@@ -47,6 +47,14 @@ cc = meson.get_compiler('c')
 
 cc = meson.get_compiler('c')
 
+# Checks for pkg-config packages
+dep_libxaw = dependency('xaw7', required: true)
+dep_libxmu = dependency('xmu', required: true)
+dep_libxt  = dependency('xt', required: true)
+dep_libx11 = dependency('x11', required: true)
+dep_xproto = dependency('xproto', required: true, version: '>= 7.0.17')
+deps = [dep_libxaw, dep_libxmu, dep_libxt, dep_libx11, dep_xproto]
+
 # Replacement for XORG_DEFAULT_OPTIONS
 if cc.has_argument('-fno-strict-aliasing')
   add_project_arguments('-fno-strict-aliasing', language: 'c')
@@ -105,14 +113,6 @@ endforeach
     description: f'Define to 1 if you have the <@h@> header file.',
   )
 endforeach
-
-# Checks for pkg-config packages
-dep_libxaw = dependency('xaw7', required: true)
-dep_libxmu = dependency('xmu', required: true)
-dep_libxt  = dependency('xt', required: true)
-dep_libx11 = dependency('x11', required: true)
-dep_xproto = dependency('xproto', required: true, version: '>= 7.0.17')
-deps = [dep_libxaw, dep_libxmu, dep_libxt, dep_libx11, dep_xproto]
 
 config_h = configure_file(output: 'config.h', configuration: conf)
 add_project_arguments('-DHAVE_CONFIG_H', language: ['c'])
